it/mail/chrome/messenger/phishing-afterload-warning-message.dtd

```diff
@@ -1,28 +1,28 @@
-<!ENTITY safeb.palm.warning.title "Suspected Web Forgery">
+<!ENTITY safeb.palm.warning.title "Sospetta falsificazione web">
 
-<!ENTITY safeb.palm.message.p1 "This page has been reported as a web forgery designed to trick users into sharing personal or financial information. Entering any personal information on this page may result in identity theft or other fraud. &#160;">
-<!ENTITY safeb.palm.message.p1.linkText "Read more &#187;">
-<!ENTITY safeb.palm.p1.linkStatusText "Read more &#133;">
+<!ENTITY safeb.palm.message.p1 "Questa pagina è stata segnalata come falsificazione web realizzata per ingannare gli utenti e spingerli a condividere informazioni personali o finanziarie. Inserire qualsiasi informazione personale su questa pagina potrebbe fraudolente un furto d'identità o altra frode. &#160;">
+<!ENTITY safeb.palm.message.p1.linkText "Continua a leggere &#187;">
+<!ENTITY safeb.palm.p1.linkStatusText "Continua a leggere &#133;">
 
-<!ENTITY safeb.palm.message.p2.start "These types of web forgeries are used in scams known as phishing attacks, in which fraudulent web pages and emails are used to imitate sources you may trust. You can find out more about ">
-<!ENTITY safeb.palm.message.p2.linkText "how &brandShortName; protects you">
-<!ENTITY safeb.palm.message.p2.end " from phishing attacks.">
+<!ENTITY safeb.palm.message.p2.start "Questi tipi di falsificazioni web sono utilizzate in truffe note come attacchi di phishing, in cui pagine web ed e-mail fraudolente sono utilizzate per imitare le risorse di cui ti potresti fidare. Puoi scoprire di più su ">
```

Comment:
CORRECTION NEEDED. Please replace the whole sentence with the following: 

+<!ENTITY safeb.palm.message.p2.start "Questi tipi di falsificazione web sono utilizzati in truffe conosciute come attacchi phishing, in cui pagine web ed e-mail fraudolente sono utilizzate per imitare risorse di cui ti potresti fidare. Puoi scoprire di più su ">